pkg/asset/installconfig/ovirt/validaton.go

```diff
@@ -125,3 +127,19 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+func validURL(val interface{}) error {
+	if urlStr, ok := val.(string); ok {
+		url, err := url.ParseRequestURI(urlStr)
+		if err != nil {
+			return errors.Errorf("The specified URL is invalid, got %s", urlStr)
+		}
+
+		if url.Scheme != "https" {
+			return errors.Errorf("The only URL scheme accepted is https, but got %s", url.Scheme)
+		}
```

Comment:
I think we can replace this with https://godoc.org/github.com/openshift/installer/pkg/validate#URIWithProtocol

Reply:
also this entire function can be made a little easier to read by early error return.

```
uri, ok := val.(string)
if !ok {
  return invalid expected string
}
return validate.URIWithProtocol(uri, "https")
```

Reply:
Hi @abhinavdahiya I didn't know about URIWithProtocol, great suggestion. I've replaced the code.